tests/ZendTest/Db/Adapter/Driver/Pgsql/ConnectionTest.php

```diff
@@ -80,4 +80,26 @@ public function testGetConnectionStringEncodeSpecialSymbol()
             $getConnectionString->invoke($this->connection)
         );
     }
+
+    /**
+     * @expectedException \Zend\Db\Adapter\Exception\InvalidArgumentException
+     */
+    public function testSetConnectionTypeException()
+    {
+        $this->connection->setType(3);
```

Comment:
This test also needs to be skipped if pgsql is not present, as it's exercising code that will check for those constants. I can make that change on merge.